packages/grafana-ui/src/components/Legend/LegendList.tsx

```diff
@@ -37,6 +37,7 @@ export const LegendList: React.FunctionComponent<LegendComponentProps> = ({
         flex-wrap: wrap;
         justify-content: space-between;
         width: 100%;
+        overflow-x: hidden;
```

Comment:
From accessibility POV I think this is not the optimal solution as it's gonna crop part of the legend label. I would suggest enabling label's text wrap for LegendList component.

Reply:
Thanks, thats a very valid POV. Removed it and added `pre-wrap`instead.